compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -460,6 +468,14 @@ void addBuiltInReferenceType(DiagnosticPos pos, Set<Whitespace> ws, String typeN
         addType(refType);
     }
 
+    void startErrorType() {
+        this.isInErrorType++;
+    }
+
+    void endErrorType() {
```

Comment:
Can we change this to private method?

Reply:
This is called from the `BLangParserListener`. So cannot make it private

Reply:
I don't see any usage of `endErrorType()` from the `BLangParserListener`. Only the `startErrorType()` is used there.

Reply:
Sorry, my bad. Yes, you are correct, can make it private.